From 4e40271658ff14ce4218147a56479e93377c05f7 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Mon, 5 Oct 2020 21:02:11 -0700
Subject: [PATCH 103/115] Deprioritize important developer notifications

Informing the user that ADB is enabled does not necessitate an alert
sound. It's especially annoying with the default AOSP notification sound,
but even with a less distracting sound, this doesn't need to be so
intrusive.

Change-Id: Idee92b44cff2473a3686a2f2d95c6f5e4e91c2f8
---
 .../internal/notification/SystemNotificationChannels.java       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/com/android/internal/notification/SystemNotificationChannels.java b/core/java/com/android/internal/notification/SystemNotificationChannels.java
index 99fe54e442c..5a8392e8442 100644
--- a/core/java/com/android/internal/notification/SystemNotificationChannels.java
+++ b/core/java/com/android/internal/notification/SystemNotificationChannels.java
@@ -103,7 +103,7 @@ public class SystemNotificationChannels {
         final NotificationChannel developerImportant = new NotificationChannel(
                 DEVELOPER_IMPORTANT,
                 context.getString(R.string.notification_channel_developer_important),
-                NotificationManager.IMPORTANCE_HIGH);
+                NotificationManager.IMPORTANCE_MIN);
         developerImportant.setBlockable(true);
         channelsList.add(developerImportant);
 
-- 
2.25.1

